<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Alışveriş Listesi</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkB_domates">
        <property name="text">
         <string>Domates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkB_biber">
        <property name="text">
         <string>Biberr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkB_patlican">
        <property name="text">
         <string>Patlıcan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkB_erik">
        <property name="text">
         <string>Erik</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkB_patates">
        <property name="text">
         <string>Patates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkB_sogan">
        <property name="text">
         <string>Soğan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkB_sarimsak">
        <property name="text">
         <string>Sarımsak</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushB_okey">
        <property name="text">
         <string>OKEY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushB_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
